tools/run_tests/run_interop_tests.py

```diff
@@ -1230,6 +1230,11 @@ def aggregate_http2_results(stdout):
       _HTTP2_TEST_CASES, http2_server_test_cases, resultset, num_failures,
       args.cloud_to_prod_auth or args.cloud_to_prod, args.prod_servers,
       args.http2_interop)
+  
+  if num_failures:
```

Comment:
Does this actually work? According to the manual sys.exit() throws SystemExit exception which seems to be eaten in except Exception as e: